openstudiocore/src/model/AirLoopHVAC.cpp

```diff
@@ -914,6 +914,25 @@ namespace detail {
     bool result = setString(OS_AirLoopHVACFields::DesignSupplyAirFlowRate, "autosize");
     OS_ASSERT(result);
   }
+  
+  /** Baseline Pressure Drop Adjustment **/
+  // Return the Baseline Pdrop
+  double AirLoopHVAC_Impl::baselinePressureDropAdjustment() const {
+    return getDouble(OS_AirLoopHVACFields::BaselinePressureDropAdjustment,true);
```

Comment:
He was correct, the boost optional should not be in the public interface.  But inside the function (which no one else can see) you do have to use boost optional, you just call .get on it before you return it to the user.  The user never sees the optional.